cypress/integration/e2e/commercial.spec.js

```diff
@@ -0,0 +1,27 @@
+import { getPolyfill } from '../../lib/polyfill';
+import { fetchPolyfill } from '../../lib/config';
+
+describe('Commercial E2E tests', function () {
+    before(getPolyfill);
+
+    beforeEach(function () {
+        // Disable CMP
+        cy.setCookie('gu-cmp-disabled', 'true', {
+            log: true,
+        });
+
+    });
+
+    describe('for WEB', function () {
```

Comment:
This describe is optional. It made sense before when we different sets of tests but you can delete it here in this context (if you like 🤷 )

Reply:
I've now added a second frontend test and have renamed the describe description